mysql-test/include/have_rocksdb_as_default.inc

```diff
@@ -0,0 +1,18 @@
+--source include/have_rocksdb.inc
+
+if (`SELECT COUNT(*) = 0 FROM INFORMATION_SCHEMA.ENGINES WHERE engine = 'rocksdb' AND support IN ('DEFAULT')`)
+{
+  --skip Test requires default engine RocksDB
+}
+
+# MYR-14
```

Comment:
As mentioned above, each of these specifically refers to a JIRA issue that needs completing before we can consider the import complete. They exist at this point solely to identify where/what needs to be done and to allow a successful compilation with reduced feature set in order to be able to begin incremental testing as things progress.